customize.html

```diff
@@ -11,56 +11,50 @@
 
 <h1 id="subgames">Subgames</h1>
 <p>
-  <a href="http://wiki.minetest.net/Subgames">Subgames</a> form a foundation for the game.  Different subgames have different objectives – such as survival, building, or player vs. player.
-  Usually Minetest comes with Minetest Game, which supplies the default items.
+  <a href="http://wiki.minetest.net/Subgames">Subgames</a> provide a foundation for
+  the game. Different subgames have different objectives – such as survival,
```

Comment:
don't you need to "escape" this unicode char in HTML?

Reply:
No, not in HTML5. Unicode characters can be used directly just fine, only `<`, `>` and `&` need to be escaped.